library/entity/multipart/src/main/java/org/quiltmc/qsl/multipart/mixin/PlayerEntityMixin.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.multipart.mixin;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.PlayerEntity;
+import org.quiltmc.qsl.multipart.api.EntityPart;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyVariable;
+
+@Mixin(PlayerEntity.class)
+public class PlayerEntityMixin {
+	@ModifyVariable(method = "attack", at=@At(value = "STORE"), ordinal = 1)
+	private Entity attackEntityPart(Entity entity) {
+		if (entity instanceof EntityPart<?> entityPart) {
+			return entityPart.getOwner();
+		}
```

Comment:
Nitpick: there should be a line break here:
```suggestion
		}

```